src/modules/create_new_involment_app.js

```diff
@@ -0,0 +1,16 @@
+const createApp = () => {
+  const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+  const config = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+
+  fetch(url, config)
+    .then((Response) => Response.text())
+    .then((data) => data)
+    .catch((error) => error);
+};
```

Comment:
- [OPTIONAL] It's one of the project requirements to use async and await with the fetch method to send and receive data. Instead of relying on Promises (using .then, .then), consider incorporating the await keyword. Kindly make the necessary adjustment. 👍🏻